pod-operation/src/main.rs

```diff
@@ -7,31 +7,39 @@ mod components;
 mod demo;
 mod state_machine;
 
+use crate::components::gyro::Mpu6050Sensor;
 use crate::components::lim_temperature::LimTemperature;
 use crate::components::pressure_transducer::PressureTransducer;
 use crate::components::signal_light::SignalLight;
 use crate::state_machine::StateMachine;
-
 #[tokio::main]
 async fn main() -> Result<(), Box<dyn std::error::Error>> {
 	tracing::subscriber::set_global_default(FmtSubscriber::default())?;
 
 	let (layer, io) = SocketIo::new_layer();
 
-	let signal_light = SignalLight::new();
-	tokio::spawn(demo::blink(signal_light));
+	// let signal_light = SignalLight::new();
+	// tokio::spawn(demo::blink(signal_light));
 
-	let pressure_transducer = PressureTransducer::new(0x40);
-	tokio::spawn(demo::read_pressure_transducer(pressure_transducer));
+	// let pressure_transducer = PressureTransducer::new(0x40);
+	// tokio::spawn(demo::read_pressure_transducer(pressure_transducer));
 
-	let ads1015 = LimTemperature::new(ads1x1x::SlaveAddr::Default);
-	tokio::spawn(demo::read_ads1015(ads1015));
+	// let ads1015 = LimTemperature::new(ads1x1x::SlaveAddr::Default);
+	// tokio::spawn(demo::read_ads1015(ads1015));
```

Comment:
Chore: let's keep the existing demos in source so the compiler doesn't complain about unused modules/functions. If you need to disable them for testing in isolation, that's okay, but avoid staging such changes.